src/index.html

```diff
@@ -27,80 +27,99 @@ <h1>What's Cookin'</h1>
         <h2>Filter</h2>
         <div class="filters">
           <div class="single-filter">
-            <input type="checkbox" name="antipasti">
-            <label for="antipasti">Antipasti</label>
+            <label for="antipasti">
```

Comment:
Doing some research- we think/hope this is going to help us grab the values we need easily so we can manipulate the DOM.

Reply:
this looks like it's heading in the right direction! I remember using the label for radio buttons a few projects ago, and followed the same setup where the label was the same as what was listed for that button (aka 'antipasti' for 'Antipasti')